src/components/modals/ConsolidateFundsModal.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+    <modal ref="modal" :title="$t('advanced.consolidate_funds.modal.title')">
+        <div class="modal_body">
+            <p>
+                {{
+                    balances?.length
+                        ? $t('advanced.consolidate_funds.modal.body')
+                        : $t('advanced.consolidate_funds.modal.no_utxos')
+                }}
+            </p>
+
+            <div class="balance_list" v-if="balances?.length">
+                <div class="balance_list_row">
+                    <div class="balance_list_item">
+                        <h3>{{ $t('advanced.consolidate_funds.modal.derived_addresses') }}</h3>
+                    </div>
+                    <div class="balance_list_item">
+                        <h3>CAM</h3>
+                    </div>
+                </div>
+                <div class="divider"></div>
+                <div class="balance_list_row" v-for="(balance, i) in balances" :key="i">
+                    <div class="balance_list_item">
+                        <p>{{ balance.addresses?.[0] }}</p>
+                    </div>
+                    <div class="balance_list_item">
+                        <p>{{ balance.amount?.toLocaleString() }}</p>
+                    </div>
+                </div>
+            </div>
+            <div class="modal_actions">
+                <v-btn @click="close" plain small>
+                    {{
+                        balances?.length
+                            ? $t('advanced.consolidate_funds.modal.cancel')
+                            : $t('advanced.consolidate_funds.modal.discard')
+                    }}
+                </v-btn>
+                <v-btn
+                    v-if="balances?.length"
+                    @click="ctaClick"
+                    class="button_primary"
+                    small
+                    depressed
+                    :loading="loading"
+                >
+                    {{ $t('advanced.consolidate_funds.modal.title') }}
+                </v-btn>
+            </div>
+            <p v-if="balances?.length" class="fee_note">
+                {{
+                    $t('advanced.consolidate_funds.modal.fee_note', {
+                        feeAmount: totalFee ?? '0.02',
```

Comment:
Why is the fallback value for the totalFee set to 0.02 instead of being dynamically determined?

Reply:
totalFee is already calculated dynamically. 0.02 is just the fallback if for some reason totalFee can't be calculated. It can be removed  as well